addons/pos_loyalty/tests/test_frontend.py

```diff
@@ -2238,3 +2238,63 @@ def test_apply_reward_on_product_scan(self):
             "PosRewardProductScanGS1",
             login="pos_admin",
         )
+
+    def test_coupon_pricelist(self):
+        tax01 = self.env["account.tax"].create({
+            "name": "C01 Tax",
+            "amount": "0.00",
+        })
+
+        self.env["product.product"].create(
+            {
+                "name": "Test Product 1",
+                "is_storable": True,
+                "list_price": 25,
+                "available_in_pos": True,
+                "taxes_id": [(6, 0, [tax01.id])],
+            }
+        )
+
+        alt_pos_config = self.main_pos_config.copy()
```

Comment:
Why are you duplicating the pos_config ? Is it not working with the existing one ?

Reply:
yes it wasn't working.